main/src/test/java/org/mobilitydata/gtfsvalidator/validator/NoticeDocumentationTest.java

```diff
@@ -77,6 +77,40 @@ public void testThatAllValidationNoticesAreDocumented() {
         .isEmpty();
   }
 
+  @Test
+  public void testThatAllValidationNoticesAreDocumentedWithFirstLine() {
+    List<Class<?>> noticesWithImproperMultilineDocComment =
+        discoverValidationNoticeClasses()
+            .filter(
+                clazz -> {
+                  NoticeDocComments docComments = NoticeSchemaGenerator.loadComments(clazz);
+                  if (docComments.getDocComment() == null) {
+                    return false;
+                  }
+                  String[] lines = docComments.getDocComment().split("\n");
```

Comment:
I think this the intended method to test: 
```suggestion
                  String[] lines = docComments.getCombinedDocumentation().split("\n");
```